### ISSUE TITLE
Post editorial activities to twitter

### ISSUE THREAD

**Issue by literarymachine** [hbz/oerworldmap#610]

**Comment by literarymachine** [comment]
Created account https://twitter.com/oerwmupdates

**Comment by philboeselager** [comment]
Favourite solution:

- create separate twitter account
- one tweet per edit
- re-tweet from main OER WM account what is important

**Comment by literarymachine** [comment]
On https://beta.oerworldmap.org/, edit there and check https://twitter.com/oerwmupdates. We might want to give the content of the tweet another thought.

**Comment by literarymachine** [comment]
Perhaps the commit (i.e. the log entry) of the update would be the right thing to link to from the tweet. #792 is a precondition for this. Once thing to keep in mind though is that commits contain the email address of the author and are thus currently only accessible by editors and admins.

**Comment by acka47** [comment]
I would completely remove the timestamp from the tweet. The timestamp of the tweet itself suffices. Maybe the tweet should both contain the link to the resource _and_ to the commit à la:

> https://oerworldmap.org/resource/resource_x was updated: http://link-to-commit.org

Thus, +1 for: 
> Perhaps the commit (i.e. the log entry) of the update would be the right thing to link to from the tweet. #792 is a precondition for this.

**Comment by literarymachine** [comment]
> I would completely remove the timestamp from the tweet.

Admittedly, I only put it there because while testing sending the same tweet multiple times was rejected.

> Maybe the tweet should both contain the link to the resource and to the commit à la

Would this include making logs / commits publicly available? IMO we would have to think about anonymizing them, i.e. remove the email then.

**Comment by acka47** [comment]
> Would this include making logs / commits publicly available? IMO we would have to think about anonymizing them, i.e. remove the email then.

Does anything speak against showing the profile URL – or just the urn – in the commit instead of the email? If people haven't added any information it won't be shown but they are nonetheless unambiguously identified..

**Comment by literarymachine** [comment]
> Does anything speak against showing the profile URL – or just the urn – in the commit instead of the email?

Currently, the account ID is used in commit messages, and this is the email address used to log in. I would like to keep it like that, but we could reformat a commit before exposing it via a URL.

**Comment by karindr** [comment]
Do you want to put a link to the profile in the tweet or do you want to link to the 'history page' like this
https://oerworldmap.org/log/urn:uuid:fea85682-1bfe-4575-8c33-8706c658d59c? I would prefer the first because the other page is a bit cryptic.

**Comment by acka47** [comment]
> Do you want to put a link to the profile in the tweet or do you want to link to the 'history page' like this
https://oerworldmap.org/log/urn:uuid:fea85682-1bfe-4575-8c33-8706c658d59c? I would prefer the first because the other page is a bit cryptic.

Just to be clear, I wouldn't link to the user profile but only to the changed resource. Yes, the commits are currently quite cryptic to normal users but it is important to show what has changed regarding a specific resource. We should add a human-readable view at some point. I made an issue for this (#895).

**Comment by trugwaldsaenger** [comment]
@literarymachine: I saw that the issues is assigned to @drrobertfarrow and me. So here comes a comment from me, though I might not be the best person to do so. I think it would be great if the tweed would be more descriptive. For example it would be helpful if the tweed would use the name of the ressources, saying something like "Humanities Open University has just been added to OER World map". For new additions I think it would make sense to link to the resource directly. If it has been only updated, it sounds reasonable to link to the history/commits/diff page.
